### DISCUSSION TITLE
New tariffs seem to have broken some things.

### DISCUSSION THREAD

**Discussion by mark-leonard-reynolds** [amberelectric/public-api#211] [General]
"Could not find BLNE23AU toGrid feedIn tariff for 4407351815"
https://app.amber.com.au/
Guessing it's going to be a busy day? :-(

**Comment by mark-leonard-reynolds** [comment]
I thought phone app was ok, until I clicked on useage tab.
"Could not find BLNE23AU toGrid feedIn tariff for SAMPLE_MARKET_ESSENTIAL _ENERGY"
Live prices & Devices tabs seem to be doing the right things though.

**Comment by madpilot** [comment]
It’s already been a busy weekend

**Reply by mark-leonard-reynolds** [reply]
I bet it has. Seems nearly OK to me, for such a big change.

**Reply by madpilot** [reply]
There is about three tariffs that are broken due to the way the networks changed them. The required changes were a bit too big for me to do over then weekend, so we’ll get a team together today to sort it out.

**Reply by mark-leonard-reynolds** [reply]
Coolio!

**Reply by mark-leonard-reynolds** [reply]
all fixed I think :-) well done :-)
